CHANGELOG.md

```diff
@@ -41,6 +41,8 @@ terraform values and it will also update the database cluster to use the new con
 
 ### Changed
 
+- **CUMULUS-3564**
+  - Update webpack configuration to explicitly disable chunking
```

Comment:
So what exactly was the issue? I trust this solves it but it would be good to verify how we got from a Tea Map Cache error to this.

Reply:
@jennyhliu  that's exactly right.    💯